Tests iOS/LoadingTests.swift

```diff
@@ -0,0 +1,27 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import XCTest
+import Sails
+import NIO
+
+class LoadingTests: XCTestCase {
+    var server: Application!
+    var app: PocketAppElement!
+    
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+        
+        let uiApp = XCUIApplication()
+        app = PocketAppElement(app: uiApp)
+    }
+    
+    override func tearDownWithError() throws {
+        app.terminate()
+    }
+    
+    func test_initialHomeState_showsLoadingState() {
+        app.launch().loadingCell.wait()
```

Comment:
Sorry, I'm not verifying this on my system, is this test making sure that the loading starts and then stop/hide after certain secs?

Reply:
just showing loading cell for now if there's nothing loaded

Reply:
decided to test with unit test instead of UI tests